### PR TITLE
Update index_en.rst

### PR DESCRIPTION
**Pull request by legend06hvl**
update en version
fix https://github.com/PaddlePaddle/Paddle/issues/8908

### CODE REVIEW

File: doc/v2/howto/index_en.rst [hunk 1]
```diff
@@ -1,11 +1,37 @@
 HOW TO
-=======
+========
+
+PaddlePaddle allows users to flexibly set various command line parameters to control the model training or forecasting process. How to use please refer to:
```
Comment (abhinavarora) [inline doc/v2/howto/index_en.rst]: PaddlePaddle provides the users the ability to flexibly set various command line parameters to control the model training and inference process. Please refer to the following instructions on using PaddlePaddle

File: doc/v2/howto/index_en.rst [hunk 2]
```diff
@@ -1,11 +1,37 @@
...
+
+..  toctree::
+  :maxdepth: 1
+
+  cmd_parameter/index_cn.rst
+
+PaddlePaddle supports distributed training tasks on fabric clusters, MPI clusters, and kubernetes clusters. For detailed configuration and usage instructions, refer to:
```
Comment (abhinavarora) [inline doc/v2/howto/index_en.rst]: kubernetes -> **Kubernetes**

File: doc/v2/howto/index_en.rst [hunk 3]
```diff
@@ -1,11 +1,37 @@
...
+
+..  toctree::
+  :maxdepth: 1
+
+  cluster/index_cn.rst
+
```
Comment (abhinavarora) [inline doc/v2/howto/index_en.rst]: the -> **a**
prediction -> **inference**

File: doc/v2/howto/index_en.rst [hunk 4]
```diff
@@ -1,11 +1,37 @@
...
+PaddlePaddle provides the C-API for prediction. We provide the following guidelines about the use of C-API:
```
Comment (sidgoyal78) [inline doc/v2/howto/index_en.rst]: "about the use of C-API" -> for using the C-API

**Comment by shanyi15** [comment]
Thank you @abhinavarora @sidgoyal78

**Pull request merged**